lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -31,6 +31,30 @@ def setup_convergence(action_handler, machine)
         chef_server_url = chef_server[:chef_server_url]
         chef_server_url = machine.make_url_available_to_remote(chef_server_url)
 
+        convergence_options[:remote_forwards].each_pair do |local_url, path_or_config|
+            options = {:force_port_forward => true}
+
+            if path_or_config.class <= Hash
+                normal_path = path_or_config[:normal_path]
+                options.merge! path_or_config
+            else
+                normal_path = path_or_config
+            end
+
+            remote_url = machine.make_url_available_to_remote(local_url, options)
+
+            parts = normal_path.split('/')
```

Comment:
Hey @causton81, I am confused - what are lines 46-56 doing?  Adding the forwarded ports to the machine_spec?